modules/freertos/11.1.0.bcr.1/patches/bazel.patch

```diff
@@ -0,0 +1,869 @@
++    name = "port_Xtensa",
++    constraint_setting = ":port",
++)
++
++constraint_value(
++    name = "port_Posix",
++    constraint_setting = ":port",
++)
++
++constraint_value(
++    name = "port_RP2040",
++    constraint_setting = ":port",
++)
++
++cc_library(
++    name = "rp2040_sdk_deps",
++    deps = [
++        "@pico-sdk//src/rp2_common/hardware_clocks",
++        "@pico-sdk//src/rp2_common/hardware_exception",
++        "@pico-sdk//src/rp2_common/pico_multicore",
++    ],
++)
++
++cc_library(
++    name = "freertos",
++    srcs = [
++        "croutine.c",
++        "event_groups.c",
++        "list.c",
++        "queue.c",
++        "stream_buffer.c",
++        "timers.c",
++    ] + select({
++        ":port_ARM_CM0": ["portable/GCC/ARM_CM0/port.c"],
++        ":port_ARM_CM3": ["portable/GCC/ARM_CM3/port.c"],
++        ":port_ARM_CM33_NTZ": [
++            "portable/GCC/ARM_CM33_NTZ/non_secure/port.c",
++            "portable/GCC/ARM_CM33_NTZ/non_secure/portasm.c",
++        ],
++        ":port_ARM_CM4F": ["portable/GCC/ARM_CM4F/port.c"],
++        ":port_ARM_CM7": ["portable/GCC/ARM_CM7/r0p1/port.c"],
++        ":port_Xtensa": [
++            "portable/ThirdParty/XCC/Xtensa/mpu.S",
++            "portable/ThirdParty/XCC/Xtensa/port.c",
++            "portable/ThirdParty/XCC/Xtensa/portasm.S",
++            "portable/ThirdParty/XCC/Xtensa/portclib.c",
++            "portable/ThirdParty/XCC/Xtensa/portmpu.c",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_context.S",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_coproc_handler.S",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_intr.c",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_intr_asm.S",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_intr_wrapper.c",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_overlay_os_hook.c",
++            "portable/ThirdParty/XCC/Xtensa/xtensa_vectors.S",
++        ],
++        ":port_Posix": [
++            "portable/ThirdParty/GCC/Posix/port.c",
++            "portable/ThirdParty/GCC/Posix/utils/wait_for_event.c",
++            "portable/ThirdParty/GCC/Posix/utils/wait_for_event.h",
++        ],
++        ":port_RP2040": [
++            "portable/ThirdParty/GCC/RP2040/port.c",
++        ],
++        "//conditions:default": [],
++    }),
++    includes = ["include/"],
++    textual_hdrs = [
++        "include/FreeRTOS.h",
++        "include/StackMacros.h",
++        "include/croutine.h",
++        "include/deprecated_definitions.h",
++        "include/event_groups.h",
++        "include/list.h",
++        "include/message_buffer.h",
++        "include/mpu_wrappers.h",
++        "include/portable.h",
++        "include/projdefs.h",
++        "include/queue.h",
++        "include/semphr.h",
++        "include/stack_macros.h",
++        "include/stream_buffer.h",
++        "include/task.h",
++        "include/timers.h",
++    ],
++    deps = [
++        ":freertos_config",
++        ":freertos_malloc",
```

Comment:
Shouldn't this now be `:freertos_heap`? Otherwise, I think the change has no effect, because the new label flag isn't referred to from anywhere.